families/Dockerfile-installed

```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 Cargill Incorporated
+# Copyright 2018 Intel Corporation
```

Comment:
Since you are not adding a new file but rewriting an old one the way we have handled Copyright in the past has been to add the new copyrrite below the previous for example

Copyright 2018 Cargill Incorporated
Copyright 2018 Intel Corporation